api/managementpb/backup/backups.proto

```diff
@@ -200,6 +201,9 @@ service Backups {
       post: "/v1/management/backup/Backups/Start"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      description: "Could return the Error message in the details containing specific ErrorCode indicating failure reason:\nERROR_CODE_XTRABACKUP_NOT_INSTALLED - xtrabackup is not installed on the service\nERROR_CODE_INVALID_XTRABACKUP - different versions of xtrabackup and xbcloud\nERROR_CODE_INCOMPATIBLE_XTRABACKUP - xtrabackup is not compatible with MySQL for taking a backup"
```

Comment:
Why it's written in one line? Can we have it in human-readable format?

Reply:
cannot find any info that proto files support multi-line literals :(

Reply:
Yep, it seems there is no such option for now: https://github.com/protocolbuffers/protobuf/issues/1297